share/smack/lib/smack.c

```diff
@@ -1006,6 +1013,24 @@ void __SMACK_decls() {
 #undef D
 }
 
+// The size parameter represents number of bits that are being accessed.
```

Comment:
Why is size parameter the number of bits?
Can it not be the number of bytes?
It is awkward to have this division by 8 in Boogie, and this might cause problems.